beater/connbeat_test.go

```diff
@@ -196,6 +208,15 @@ func TestNoContainerInformationLeakage(t *testing.T) {
 	assert.Nil(t, container, "Container information should not leak into the second event")
 }
 
+func expectMap(t *testing.T, actual common.MapStr, expected map[string]string) {
```

Comment:
nice - might be good to swap around `expected` and `actual`/`got` to be more consistent with github.com/stvp/assert? (I know, I should also have done that or `expectElements` :))